sql/core/src/main/scala/org/apache/spark/sql/jdbc/OracleDialect.scala

```diff
@@ -29,7 +29,12 @@ private case object OracleDialect extends JdbcDialect {
   override def getCatalystType(
```

Comment:
Aha, I see. you're right and it seems `JdbcUtils` always sets `scale` https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala#L285.
However, since I got some reports (https://www.mail-archive.com/user@spark.apache.org/msg61280.html) from spark-users, I made this pr. I feel some weird and I keep looking into this. I appreciate your suggestions.